### PR TITLE
fix: change for issue #36

### PR DESCRIPTION
**Pull request by RobinTheHood**

### CODE REVIEW

File: src/admin/includes/extra/javascript/rth_stripe.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * https://docs.module-loader.de
+ *
+ * @author  Robin Wieschendorf <mail@robinwieschendorf.de>
+ * @author  Jay Trees <stripe@grandels.email>
+ * @link    https://github.com/RobinTheHood/modified-stripe/
+ */
+
+namespace RobinTheHood\Stripe;
+
+require_once DIR_FS_DOCUMENT_ROOT . '/includes/extra/functions/rth_modified_std_module.php';
```
Comment (grandeljay) [inline src/admin/includes/extra/javascript/rth_stripe.php:16]: This line is not required. The Stripe module has the std-module as a dependency which already handles auto loading for us.

Could you remove it please?

File: src/admin/includes/extra/javascript/rth_stripe.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+if (rth_is_module_disabled(Classes\Constants::MODULE_PAYMENT_NAME)) {
+    return;
+}
+
+?>
+<script>
+    document.addEventListener('DOMContentLoaded', function() {
+        const tabs = document.querySelectorAll('.rth-stripe-tabs');
+        tabs.forEach(function(tab) {
+            // console.log(tab);
```
Comment (grandeljay) [inline src/admin/includes/extra/javascript/rth_stripe.php:27]: While you're at it, can you also please remove all `console.log`'s?

**Review by grandeljay — CHANGES_REQUESTED** [review]

**Comment by RobinTheHood** [comment]
Thanks for your review. I have processed your comments and made two new commits.

**Review by grandeljay — APPROVED** [review]
LGTM

**Pull request merged**